src/slack.coffee

```diff
@@ -214,21 +205,12 @@ class SlackBot extends Adapter
     text
 
   send: (envelope, messages...) ->
-    channel = @client.getChannelGroupOrDMByName envelope.room
-    channel = @client.getChannelGroupOrDMByID(envelope.room) unless channel
-
-    if not channel and @client.getUserByName(envelope.room)
-      user_id = @client.getUserByName(envelope.room).id
-      @client.openDM user_id, =>
-        this.send envelope, messages...
-      return
-
```

Comment:
Nope. I cut it because we're send from the client now, rather than [then channel object](https://github.com/slackhq/hubot-slack/pull/309/files/cd61c608b2084759c51c95ed0777208a1e91c9e1#diff-44aca805fd3e634b2ed4f6bc2ccb0749L242). I didn't see where else we were using channel there and figured it could go. 

Very possible I missed something though.